src/client/common/client_common.cpp

```diff
@@ -145,9 +145,7 @@ mp::ReturnCode mp::cmd::standard_failure_handler_for(const std::string& command,
                                                      const grpc::Status& status, const std::string& error_details)
 {
     fmt::print(cerr, "{} failed: {}\n{}", command, status.error_message(),
-               !error_details.empty()
-                   ? fmt::format("{}\n", error_details)
-                   : !status.error_details().empty() ? fmt::format("{}\n", status.error_details()) : "");
```

Comment:
Good find, although this returns a string, it has [only *binary* details](https://grpc.github.io/grpc/cpp/classgrpc_1_1_status.html#a3eb2cecab0655d1f74c191847332824d).

Reply:
Hey @ricab, thanks for taking a look to this. What do you mean with this? The output is indeed correct now.